### ISSUE TITLE
TeamCity dependencies plugin requires Gradle 5.3. Gradle 6.1-rc-2 detected.

### ISSUE THREAD

**Issue by chrisg123** [fwcd/kotlin-language-server#183]
Gradle 6.1-rc2 allows for Gradle's Dependency cache to be relocated. This is helpful for offline builds [(ref)](https://docs.gradle.org/6.1-rc-2/release-notes.html#gradle's-dependency-cache-can-be-relocated).

Trying to build kotlin-language-server with Gradle 6.1-rc2 however results in the following error:

> TeamCity dependencies plugin requires Gradle 5.3. Gradle 6.1-rc-2 detected.

TeamCity-dependencies-gradle-plugin v1.2  Fixes compatibility with gradle 6.0 [(ref)](https://github.com/jk1/TeamCity-dependencies-gradle-plugin/commit/f23882998640a8d33359a2c98ddab49538a384a2?diff=unified) 

The build also still works with Gradle 5.3 after the following changes.

```
diff --git a/server/build.gradle b/server/build.gradle
index 9946837..f7d35a1 100644
--- a/server/build.gradle
+++ b/server/build.gradle
@@ -2,7 +2,7 @@ plugins {
     id 'org.jetbrains.kotlin.jvm'
     id 'maven'
     id 'application'
-       id 'com.github.jk1.tcdeps' version '0.19'
+       id 'com.github.jk1.tcdeps' version '1.2'
     id 'com.jaredsburrows.license' version '0.8.42'
 }
```
Happy to submit a pull request if this makes sense.

**Comment by fwcd** [comment]
Yeah, a PR would be good!

**Comment by fwcd** [comment]
Fixed with #184.
